realsense2_camera/srv/MapPathString.srv

```diff
@@ -0,0 +1,3 @@
+string filepath
+---
+bool success
```

Comment:
the service message definition. It takes as input a filepath, and returns `success = true` if it succeeds to save the map.